addons/breathing/stringtable.xml

```diff
@@ -2748,50 +2748,62 @@
         <Key ID="STR_KAT_Breathing_SETTING_AVAIL_OXYMASK">
             <English>Available Oxygen Masks</English>
             <Japanese>使用可能な酸素マスク</Japanese>
+            <French>Masques à oxygène disponibles</French>
         </Key>
         <Key ID="STR_KAT_Breathing_SETTING_AVAIL_OXYMASK_DISC">
             <English>Sets which goggles can act as oxygen masks</English>
             <Japanese>酸素マスクとして動作させたいゴーグルを指定します</Japanese>
+            <French>Spécifiez les lunettes que vous souhaitez utiliser comme masque à oxygène.</French>
```

Comment:
Spécifiez les masques que vous souhaitez utiliser comme masque à oxygène.

(Here we make reference to the items in the arsenal, imo "masques" is less confusing)